engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreEngineActor.scala

```diff
@@ -146,8 +146,9 @@ object WorkflowStoreEngineActor {
   sealed trait WorkflowStoreEngineActorResponse
   case object NoNewWorkflowsToStart extends WorkflowStoreEngineActorResponse
   final case class NewWorkflowsToStart(workflows: NonEmptyList[WorkflowToStart]) extends WorkflowStoreEngineActorResponse
-  final case class WorkflowAborted(workflowId: WorkflowId) extends WorkflowStoreEngineActorResponse
-  final case class WorkflowAbortFailed(workflowId: WorkflowId, reason: Throwable) extends WorkflowStoreEngineActorResponse
+  sealed abstract class WorkflowStoreEngineAbortResponse extends WorkflowStoreEngineActorResponse
```

Comment:
I've preferred abstract classes to traits where possible for a long time now (mostly for reasons cited by [this SO answer](https://stackoverflow.com/a/35251513)) although it's never bubbled up to be a thing that I badger others about because at the end of the day it's a fairly minor point IMO.

Also it's trivial to change to a `Trait` if it ever becomes necessary.